File: langs/file.cpp

#include <fstream>
#include <iostream>
using namespace std;

int main()
{

    const string filename = "file.txt";
    // check file

    ifstream f1(filename);

    if (f1.good())
    {
        cout << filename << " exist " << endl;
    }
    else
    {
        cout << filename << " does not exist " << endl;
    }

    
    return 0;
}

File: langs/file.py
import os

# check file exsit
if os.path.exists("file.txt"):
    print(" file.txt exist ")
else:
    print("file.txt does not exist!")

# read file
with open("file.txt", "r") as f:
    lines = f.readlines()
for index, line in enumerate(lines):
    print(index, line)

# write file
with open("file.txt", "a") as f:
    f.write("\nabc")
